en/docs/guides/mfa/mfa-for-management-console.md

```diff
@@ -0,0 +1,100 @@
+# Configuring multi-factor authentication for the management console
+
+ This guide walks you through configuring multi-factor authentication for the Identity Server's Management Console.
+
+ MFA configuration for the management console contains two configuration sections as follows:
+
+1. [Configure the service provider](#configure-the-service-provider)
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
```

Comment:
In this case, the two sub points are not sequential yeah? Then, they can be bullets and not numbers.